dapps/admin/src/pages/users/UserProfile.js

```diff
@@ -1,6 +1,35 @@
 import React from 'react'
 import { Tag } from '@blueprintjs/core'
-import { mapVerifiedAttestations } from '@origin/marketplace/src/utils/profileTools'
+
+const getDisplayName = provider => {
+  // TBD: Should this be translated?
```

Comment:
Translations are in profileUtils?

Reply:
Yes, We can just import that function from marketplace DApp. Just wanted to make sure if we translate things in admin DApp too.

Reply:
Ooh missed this was in admin app. No need to translate there.